cmd/soroban-cli/src/commands/global.rs

```diff
@@ -11,27 +11,27 @@ pub struct Args {
     pub locator: config::locator::Args,
 
     /// Filter logs output. To turn on `stellar_cli::log::footprint=debug` or off `=off`. Can also use env var `RUST_LOG`.
-    #[arg(long, short = 'f')]
+    #[arg(long, short = 'f', global = true)]
     pub filter_logs: Vec<String>,
 
     /// Do not write logs to stderr including `INFO`
-    #[arg(long, short = 'q')]
+    #[arg(long, short = 'q', global = true)]
     pub quiet: bool,
 
     /// Log DEBUG events
-    #[arg(long, short = 'v')]
+    #[arg(long, short = 'v', global = true)]
     pub verbose: bool,
 
     /// Log DEBUG and TRACE events
-    #[arg(long, visible_alias = "vv")]
+    #[arg(long, visible_alias = "vv", global = true)]
     pub very_verbose: bool,
 
     /// List installed plugins. E.g. `stellar-hello`
-    #[arg(long)]
+    #[arg(long, global = true)]
```

Comment:
List doesn't make much sense being global. It's intended for getting a list of plugins, which is something that doesn't make much sense on sub commands.